build.psm1

```diff
@@ -2377,6 +2377,7 @@ function Start-CrossGen {
             "Microsoft.PowerShell.CoreCLR.Eventing.dll",
```

Comment:
@iSazonov, your last commit had 7 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Validate simple Set-LocalGroup.Can Set-LocalGroup by name
```powershell
Expected exactly 'Test Group Set 1 new description', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroup.Tests.ps1: line 392
392:             $result.Description | Should -BeExactly "Test Group Set 1 new description"
```
Validate simple Rename-LocalGroup.Can Rename-LocalGroup using a valid group name
```powershell
Expected strings to be the same, but they were different.
Expected length: 17
Actual length:   16
Strings differ at index 16.
Expected: 'TestGroupRename1x'
But was:  'TestGroupRename1'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroup.Tests.ps1: line 503
503:             $result.Name | Should -BeExactly TestGroupRename1x
```
Validate simple Add-LocalGroupMember.Can add local user to group by name
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroupMember.Tests.ps1: line 90
90:             $result.Name.EndsWith("TestUser1") | Should -BeTrue

```
Validate simple Get-LocalGroupMember.Can get a local group member by name
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroupMember.Tests.ps1: line 265
265:             $result.Name.EndsWith("TestUserGet1") | Should -BeTrue

```
Validate simple New-LocalUser.Can create New-LocalUser with only name
```powershell
Expected exactly 'TestUserNew1', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalUser.Tests.ps1: line 95
95:             $result.Name | Should -BeExactly TestUserNew1
```